android/src/main/java/com/reactnativecommunity/toolbarandroid/ReactToolbar.java

```diff
@@ -115,6 +116,8 @@ public IconImageInfo(int width, int height) {
       mHeight = height;
     }
 
+    public Map<String,Object> getExtras() { return new HashMap<String,Object>(); }
```

Comment:
a couple of things:

1. there is already a `getExtras()` method below
2. we probably don't want to keep creating HashMap instances per invocation. Let's create a private member for it instead.

Reply:
Hey @connectdotz I think that https://github.com/react-native-toolbar-android/toolbar-android/issues/57 is actually resolved by this commit https://github.com/react-native-toolbar-android/toolbar-android/commit/e5906f5a2b19396b9ddbfe765c757106767194cd

Which includes this change. 
```
    // to be compatible with RN-0.65/fresco-2.5
    public Map<String, Object> getExtras() {
      return null;
    }
```

For some reason my codebase had `toolbar-android` pinned to version `1.0-rc2` even though we aren't using the package directly. I'm going to close this pr and i think #57 can be closed with the resolution of using `v0.2.1`. Other users may need to specify version in `package.json` directly as a `dependency` or a `resolution`.